.github/resources/config-for-integration-test.json

```diff
@@ -53,8 +53,11 @@
             "port": 32331,
             "flags": [
               "--rpc-cors=all",
+              "--execution=wasm",
```

Comment:
Because #701 has been merged, perhaps the param `execution` is unnecessary any more.
Not 100% sure.

Reply:
I guess it's unnecessary but I wanted to be explicit here.